.gitignore

```diff
@@ -0,0 +1,294 @@
+### Java template
```

Comment:
I'm curious, did you use a set of templates for this?

Reply:
Yes, I am using a plugin provided by Jetbrains.